en/simulation/ignition_gazebo.md

```diff
@@ -56,12 +56,127 @@ In order to run the simulation without running the ignition gazebo gui, one can
 HEADLESS=1 make px4_sitl gz_x500
 ```
 
-In order to increase the verbose output, `VERBOSE_SIM=1` can be used:
+## Usage/Configuration Options
+
+The startup pipeline allows for highly flexible configuration.
+In particular, it is possible to:
+
+- Start a new Ignition simulation with an arbitrary world or attach to an already running simulation.
+- Add a new vehicle to Ignition or link a new PX4 instance to an existing one.
+
+These scenarios are managed by setting the appropriate environment variables.
+
+### Syntax
+
+The startup syntax takes the form:
 
 ```bash
-VERBOSE_SIM=1 make px4_sitl gz_x500
+ARGS ./build/px4_sitl_default/bin/px4
 ```
 
+where `ARGS` is a list of environment variables including:
+
+- `PX4_SYS_AUTOSTART` (**Mandatory**):
+  Sets the [airframe autostart id](../dev_airframes/adding_a_new_frame.md) of the PX4 airframe to start.
+  - Only `4001` (x500 quadcopter) is currently supported.
+
+- `PX4_GZ_MODEL_NAME`:
+  Sets the name of an _existing_ model in the gazebo simulation.
+  If provided, the startup script tries to bind a new PX4 instance to the Ignition resource matching exactly that name.
+  - The setting is mutually exclusive with `PX4_GZ_MODEL`.
+
+- `PX4_GZ_MODEL`:
+  Sets the name of a new Ignition model to be spawned in the simulator.
+  If provided, the startup script looks for a model in the Ignition resource path that matches the given variable, spawns it and binds a new PX4 instance to it.
+  - The setting is mutually exclusive with `PX4_GZ_MODEL_NAME`.
+
+  :::note
+  If both `PX4_GZ_MODEL_NAME` and `PX4_GZ_MODEL` are not given, then PX4 looks for `PX4_SIM_MODEL` and uses it as an alias for `PX4_GZ_MODEL`.
+  However, this prevents the use of `PX4_GZ_MODEL_POSE`.
+  :::
+
+- `PX4_GZ_MODEL_POSE`:
+  Sets the spawning position and orientation of the model when `PX4_GZ_MODEL` is adopted.
+  If provided, the startup script spawns the model at a pose following the syntax `"x,y,z,roll,pitch,yaw"`, where the positions are given in metres and the angles are in radians.
+  - If omitted, the zero pose `[0,0,0,0,0,0]` is used.
```

Comment:
Where is 000 in the model? Centre of a box? Top left front?

Reply:
`PX4_GZ_MODEL_POSE` sets the pose of the link `base_link` of the model in the world reference frame. I woluld expect  that that link is in the middle of the model, very close to the center of mass. This is actually the case for `x500`. The orientation instead follows the ENU / FLU conventions (which is different from the ones used by PX4)

Reply:
Thanks @beniaminopozzan  - leaving this item and https://github.com/PX4/PX4-user_guide/pull/2198#discussion_r1049030741 to you to do as a post-process PR